extensions/custom-footer/shopify.extension.toml

```diff
@@ -0,0 +1,30 @@
+# Learn more about configuring your checkout UI extension:
+# https://shopify.dev/api/checkout-extensions/checkout/configuration
+
+# The version of APIs your extension will receive. Learn more:
+# https://shopify.dev/docs/api/usage/versioning
+api_version = "2024-01"
+
+[[extensions]]
+type = "ui_extension"
+name = "custom-footer"
+handle = "custom-footer"
+
+# Controls where in Shopify your extension will be injected,
+# and the file that contains your extension’s source code. Learn more:
+# https://shopify.dev/docs/api/checkout-ui-extensions/unstable/extension-targets-overview
+
+[[extensions.targeting]]
+module = "./src/Footer.jsx"
+target = "purchase.checkout.footer.render-after"
+export = "checkoutFooter"
```

Comment:
Do you know where we implement this "multiple exports from a single file" feature in the CLI? In documentation, we always show one target per file (https://shopify.dev/docs/api/checkout-ui-extensions/unstable/configuration#targets-supporting-multiple-extension-targets), and I thought that was the only way it worked.

Reply:
I've seen this pattern used for other extensions, had not realized it isn't documented for ui extensions. Looking around the dev docs I did find that it's documented for [Shopify Functions extensions](https://shopify.dev/docs/apps/app-extensions/configuration#shopify-functions-extensions).

Reply:
We did implement it for functions, but I did not think that implementation applied to UI extensions. Does this example actually work if you publish it as a production extension?

Reply:
I was able to confirm the production build works on my partner dev shop.